scripts/test.sh

```diff
@@ -39,6 +39,12 @@ function run_checks_pr {
   fi
 }
 
+# Scan TLDR for vulnerabilities
+function scan_for_vulns {
+  npm install -g auditjs
```

Comment:
Instead of installing globally, you could just do `npx auditjs@latest ossi` and it will effectively do the same thing.

Reply:
good idea, i'll change that @DarthHater

Reply:
Done in https://github.com/tldr-pages/tldr/pull/3924/commits/36a864a1e2670069f425888e13c805bcac4e3c00